guides/common/modules/con_prerequisites-for-configuring-project-with-keycloak-quarkus-authentication.adoc

```diff
@@ -1,24 +1,20 @@
 [id="prerequisites-for-configuring-{project-context}-with-keycloak-authentication_{context}"]
 = Prerequisites for configuring {Project} with {keycloak-quarkus} authentication
 
-Before configuring {Project} with {keycloak-quarkus} external authentication, ensure that you meet the following requirements:
-
-* A working installation of {keycloak-quarkus} server that uses HTTPS instead of HTTP.
-* Configuring Quarkus-based {keycloak-quarkus} authentication requires the `--http-relative-path=/auth` context path to be configured in your {keycloak-quarkus} system setup.
-If you want to use a different context path, make manual adjustments after the initialisation with `/auth` or fully set up the `_foreman-openidc_oidc_keycloak_Foreman_Realm_.conf` file of the `httpd` service manually.
-+
-For more information, see {RHDocsBaseURL}red_hat_build_of_keycloak/24.0/html-single/server_guide/index#reverseproxy-different-context-path-on-reverse-proxy[Different context-path on reverse proxy] in the _{keycloak-quarkus} Administration Guide_.
-* A {keycloak-quarkus} account with admin privileges.
-* A realm for {Project} user accounts created in {keycloak-quarkus}.
-* If the certificates or the CA are self-signed, ensure that they are added to the end-user certificate trust store.
-* Users imported or added to {keycloak-quarkus}.
+* A {keycloak} account with administrative privileges.
+* A {keycloak} server that uses HTTPS instead of HTTP and has been initialized with the `--http-relative-path=/auth` context path.
+To access a {keycloak} server initialized with `--http-relative-path=/auth` from its web UI, go to `https://_{keycloak-example-com}_:8443/auth`.
```

Comment:
The 8443 was added because the demo system was running on that port, right? Because most other links in the guide have it set to the 443 default.
Or did the default change between Wildfly and Quarkus?
I actually don't know anymore, we always have the Keycloaks running on 443.

Reply:
On our side, it looked like the default for Quarkus was indeed changed to 8443.

Reply:
Of course default https port is 443 but in the Quarkus docs, I always see 8443. But that's not important, the point is to remind the user to enter the port whatever it is.